betelgeuse/parser.py

```diff
@@ -187,3 +187,31 @@ def parse_docstring(docstring=None):
             field_value = output
             fields_dict[field_name] = field_value
     return fields_dict
+
+
+def parse_markers(all_markers=None, config=None):
+    """Parse the markers."""
+    resolved_markers = []
+    ignore_list = getattr(config, 'MARKERS_IGNORE_LIST', None)
```

Comment:
```suggestion
    ignore_list = getattr(config, 'MARKERS_IGNORE_LIST', [])
```
if you simply defaulted to [] here, it could save you from writing complex expression in the conditional at line no. 202.

you could simply iterate through the list unconditionally